tests/test_ddc_overwrite_manager.c

```diff
@@ -0,0 +1,53 @@
+#include <limits.h>
+
+#include "manager.h"
+
+/* Test wrapper for compartments with arguments.
+ *
+ * Takes at least two arguments: one argument for the compartment binary name,
+ * and one argument minimum to be passed to the compartment itself.
+ * TODO currently hard coding the entry points and which entry point to call,
+ * but plan to move this to some configuration file in the near future
+ */
+
+extern struct Compartment* loaded_comp;
+
+int
+main(int argc, char** argv)
+{
+    // Initial setup
+    manager_ddc = cheri_ddc_get();
+    setup_intercepts();
+
+    char* file = "test_ddc_overwrite";
+    size_t entry_point_count = 0;
+    struct ConfigEntryPoint* cep = parse_compartment_config(file, &entry_point_count);
+    if (!cep)
+    {
+        cep = malloc(sizeof(struct ConfigEntryPoint));
+        cep = set_default_entry_point(cep);
+    }
+
+    struct Compartment* arg_comp = comp_from_elf(file, cep, entry_point_count);
+    loaded_comp = arg_comp; // TODO
```

Comment:
TODO... what?

Reply:
This is a leftover from me copy-pasting from the other managers (related to multi-compartment support). Unneeded for this test, so I just removed it in https://github.com/capablevms/CHERI-ELF-comp/pull/12/commits/3419b3e08d4c3d108e9c7c09e88af761ce65ad7c.